### PR TITLE
Add a PR template

### PR DESCRIPTION
**Pull request by Fris0uman**
Add A PR template to normalise all our PRs and maybe setup some automatised process to build changelogs

### CODE REVIEW

**Comment by int-ua** [comment]
Can we have a separate simpler version for sprite-only additions? Testing section would always be the same, no need to repeat the same text, maybe no Additional context section either?

Also, this main template needs an Infrastructure category.

**Comment by int-ua** [comment]
Just in case: I do not oppose this change. I do not particularly like it either as it will increase the amount of work I need to do but I can deal with it.

**Comment by Fris0uman** [comment]
Additional context is optional it's just to add stuff if you it's needed.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -17,17 +17,18 @@ Do not enter the square brackets [].  Tileset must be one of these:
...
 
 #### Content of the change
 
-<!-- Explain what does this pull request contains.  -->
+<!-- Explain what does this pull request contain.  -->
 
 #### Testing
 
-<!-- Describe what steps you took to test that this PR resolved the bug or added the feature, and what tests you performed to make sure it didn't cause any regressions.  Also include testing suggestions for reviewers and maintainers.  -->
+<!-- Describe what steps you took to test this PR, if applicable include screenshot of the sprites in game or at least in editor.  -->
```
Comment (int-ua) [inline .github/pull_request_template.md:30]: Maybe "image editor"? I was confused for a second :)

**Comment by int-ua** [comment]
What if we made a short documentation file that could be linked from the template and leave only one comment to remove, with one sentence and the link?

**Comment by int-ua** [comment]
I meant only comments, leaving the headings in place. And maybe not one sentence but put the whole description in one comment so it would be easier to remove in one action, not having to repeat the same thing for every heading.

**Comment by Fris0uman** [comment]
Comments are not visible, so there's actually no reason to remove them

**Comment by Fris0uman** [comment]
Basically if PRs could be:
```
SUMMARY: [Category] "[Briefly describe the change in these quotation marks]"

>List of added sprites

>Screenshots of those sprites
```
It would make them pretty easy to review

**Comment by int-ua** [comment]
> Comments are not visible, so there's actually no reason to remove them

hmm. I guess you are technically correct. It's me who considers not removing them "untidy"/non-neat.

**Comment by int-ua** [comment]
What are your thoughts on moving summary to the PR title?

**Comment by Fris0uman** [comment]
Most of the times I put exactly the same thing as summary and as title so it's fine with me. But I don't see how to put that in the template

**Comment by int-ua** [comment]
> Most of the times I put exactly the same thing as summary and as title so it's fine with me. But I don't see how to put that in the template

Yeah, looks like it's impossible currently: https://github.com/isaacs/github/issues/1252

**Comment by int-ua** [comment]
https://github.com/Fris0uman/CDDA-Tilesets/pull/3

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!-- HOW TO USE: Under each "#### Heading" below, enter information relevant to your pull request.
+Comment blocks, surrounded with <!–– and ––>, won't be visible in the actual post.-->
+
+#### Summary
+
+SUMMARY: Category "Brief description"
```
Comment (int-ua) [inline .github/pull_request_template.md:6]: What if we get rid of the literal `SUMMARY:` and leave just `Category "Brief description"`?

Comment (int-ua) [reply .github/pull_request_template.md:6]: The fact that it's a summary is already conveyed by the section title, isn't it?

**Comment by int-ua** [comment]
Oh, I executed wrong `git push` as I was trying to create a PR for your branch. I hope you are okay with these changes?

**Review by int-ua — APPROVED** [review]

**Pull request merged**